server/jetstream_test.go

```diff
@@ -16678,24 +16678,23 @@ func TestJetStreamLimits(t *testing.T) {
 		require_Equal(t, err.Error(), "duplicates window can not be larger then server limit of 1m0s")
 
 		ci, err := js.AddConsumer("foo", &nats.ConsumerConfig{Durable: "dur1", AckPolicy: nats.AckExplicitPolicy})
-		require_NoError(t, err)
-		require_True(t, ci.Config.MaxAckPending == 1000)
-		require_True(t, ci.Config.MaxRequestBatch == 250)
+		require_Error(t, err)
+		require_Equal(t, err.Error(), "consumer max request batch needs to be specified")
```

Comment:
If you accept my change, then would need to update this expected error message here.